libbeat/beat/beat.go

```diff
@@ -113,7 +113,7 @@ type Beat struct {
 type BeatConfig struct {
 	Output  map[string]*common.Config
 	Logging logp.Logging
-	Shipper publisher.ShipperConfig
+	Shipper publisher.ShipperConfig `config:",inline"`
```

Comment:
whenever I update some config struct I try to add config tags for naming (even if it feels redundant). Can we add config tags with names here?